content/about.md

```diff
@@ -28,49 +28,56 @@ cover:
 
 ### Work-related interests:
 
-- Currently designing a commercial cubesat platform at [SatRevolution](https://satrevolution.com/)
-- Playing around with UAVs - MavLink, ArduPilot & DroneKit
-- Vice president of the [Academic Aviation Club](http://akl.pwr.edu.pl)
-- Writing automation & testing scripts mainly in Python & Bash
+- Currently designing & operating a commercial CubeSat constellation @[SatRevolution](https://satrevolution.com/)
+- Playing around with UAVs - ArduPilot, computer vision, telemetry systems
+- Tech lead of the [Academic Aviation Club](http://akl.pwr.edu.pl) (2nd place [Sae Aero Design Advanced](https://www.saeaerodesign.com/) winner)
 - Extensive Linux knowledge (long time power user, ricing enthusiast)
-- Solid understanding of C++ and low-level, planning to learn Rust
-- Web services and web apps with Flask and Django
+- Solid understanding of mission operations & telemetry system design
+- Python + Rust + Bash = ❤️
 
 ### Other interests:
 
 - Hard sci-fi
-- Lots of music, weird sounds
-- Random philosophical ideas
+- Overly pretentious music
```

Comment:
meh

Reply:
Changed to something less childish, please take a look

Reply:
karr